### PR TITLE
enh(ui): display link to regex101 in search tooltip

### PR DESCRIPTION
**Pull request by kduret**
## Description

image

**Fixes** MON-5328

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [x] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 2.8.x
- [ ] 18.10.x
- [ ] 19.04.x
- [ ] 19.10.x
- [x] 20.04.x (master)

<h2> How this pull request can be tested ? </h2>

Check search tooltip in events view page

### CODE REVIEW

**Comment by kduret** [comment]
@jdelpierre did you build translation (to json format) ?

**Comment by jdelpierre** [comment]
> @jdelpierre did you build translation (to json format) ?

I tested with the docker image generated on this Pull Request.

File: www/front_src/src/Resources/SearchHelpTooltip.tsx [hunk 1]
```diff
@@ -76,6 +78,13 @@ const Content = ({ onClose }: ContentProps): JSX.Element => {
             {` ${labelSearchByHostNameStartingWith} "FR20" ${labelSearchAndHostAliasContaining} "prod"`}
           </li>
         </ul>
+        <i>
+          <b>{`${labelTips}: `}</b>
+          {`${labelGetRegexHelp} `}
+          <Link href="https://regex101.com" target="_blank">
```
Comment (Thebarda) [inline www/front_src/src/Resources/SearchHelpTooltip.tsx:84]: There is a lack of security here (about [Attack Surface](https://en.wikipedia.org/wiki/Attack_surface))
You can add `rel="noopener"` or `rel="noreferrer"`
More explanation [here](https://developers.google.com/web/tools/lighthouse/audits/noopener)

**Comment by jdelpierre** [comment]
I tested and it works well :)

**Pull request merged**